cves/kernel/CVE-2013-5634.yml

```diff
@@ -55,7 +55,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerability occured in the Linux kernel (before version 3.10, this file has
+  changed locations) and allowed host users of the OS to cause a denial of service due
+  to the vCPU not being properly initialized.
```

Comment:
I think a brief explanation of what vCPU is would help a lot here

Reply:
Good point, I'll update the description.